### PR TITLE
src: Add an API to get the BSSID.

### LINKED ISSUE #55: Add an API to get the current BSSID
So I can test a fix for https://github.com/georgerobotics/cyw43-driver/issues/54

### PR DESCRIPTION
**Pull request by peterharperuk**
Fixes #55

### CODE REVIEW

File: src/cyw43.h [hunk 1]
```diff
@@ -348,6 +348,15 @@ int cyw43_wifi_leave(cyw43_t *self, int itf);
  */
 int cyw43_wifi_get_rssi(cyw43_t *self, int32_t *rssi);
 
+/*!
+ * \brief Get the BSSID of the connected wifi network
+ *
+ * \param self the driver state object. This should always be \c &cyw43_state
+ * \param bssid a pointer to a buffer to return the mac address.
```
Comment (dpgeorge) [inline src/cyw43.h:355]: It might be better worded as "an array", rather than "a pointer".

File: src/cyw43_ll.h [hunk 2]
```diff
@@ -266,6 +266,7 @@ int cyw43_ll_wifi_scan(cyw43_ll_t *self, cyw43_wifi_scan_options_t *opts);
 int cyw43_ll_wifi_join(cyw43_ll_t *self, size_t ssid_len, const uint8_t *ssid, size_t key_len, const uint8_t *key, uint32_t auth_type, const uint8_t *bssid, uint32_t channel);
 void cyw43_ll_wifi_set_wpa_auth(cyw43_ll_t *self);
 void cyw43_ll_wifi_rejoin(cyw43_ll_t *self);
+int cyw43_ll_wifi_bssid(cyw43_ll_t *self_in, uint8_t *bssid);
```
Comment (dpgeorge) [inline src/cyw43_ll.h:269]: Please name it `cyw43_ll_wifi_get_bssid()`.

**Comment by peterharperuk** [comment]
oops - pushed the channel change to the driver, now fixed.

**Comment by dpgeorge** [comment]
Thanks!

**Pull request merged**